packages/ui-app/src/AddressInfo.tsx

```diff
@@ -79,19 +79,28 @@ class AddressInfo extends React.PureComponent<Props> {
         {balanceDisplay.bonded && (
           <>
             <Label label={t('bonded')} />
-            <Bonded
-              className='result'
-              params={value}
-            />
+            <div className='result'>
+                {staking_info && staking_info.stakingLedger
+                ? formatBalance(staking_info.stakingLedger.active)
```

Comment:
Funky indent here, extra 2 spaces before `?` and `:`

Reply:
Oops, missed that one :)